python_modules/dagster/dagster/core/workspace/context.py

```diff
@@ -283,7 +283,7 @@ class IWorkspaceProcessContext(ABC):
     """
 
     @abstractmethod
-    def create_request_context(self) -> BaseWorkspaceRequestContext:
+    def create_request_context(self, origin=None) -> BaseWorkspaceRequestContext:
```

Comment:
what's the range of possible types of this? Possible to add a comment/more explanation (particularly since 'origin' already has a long line of meanings in dagster

This is very possibly still a (small) breaking change in internal